src/entities/StartOptions.ts

```diff
@@ -1,8 +1,10 @@
 import { Account } from './Account';
+import { ConnectStyles } from './ConnectStyles';
 
 export interface StartOptions {
     sessionToken: string;
     apiUrl?: string;
+    styles?: ConnectStyles;
```

Comment:
I don't know if I like the `styles` name for this, but TBH, I don't have a better suggestion.